kythe/go/indexer/testdata/basic/anonymous.go

```diff
@@ -51,6 +51,25 @@ var w struct {
 //- @X ref X
 var _ = w.X
 
+var y struct {
+	//- @Y defines/binding Y
+	//- Y.node/kind variable
+	//- Y.subkind field
+	Y int
+} = struct {
+	//- @Y defines/binding Y2
+	//- !{ @Y defines/binding Y
+	//-    @Y ref Y }
+	Y int
+	// TODO(schroederc): investigate joining the duplicate anon types
```

Comment:
Technically they're different types that just happen to be assignable because at least one of them is not named. Not that you couldn't choose to unify them, but it is likely to be more trouble than it's worth (and may not be "correct").